### DISCUSSION TITLE
Regional-exclusivity of Yoodles

### DISCUSSION THREAD

**Discussion by upintheairsheep** [ftde0/yt2009#190] [Ideas]
Since 2007, YouTube had Yoodles, basically Google Doodles but for YouTube, on specific days and holidays, some restricted to certain countries. yt2009 can implement the 2009 logos on certain days.

https://logos.fandom.com/wiki/YouTube/Yoodles

**Comment by ftde0** [comment]
already done.

https://github.com/ftde0/yt2009/tree/main/assets/site-assets/doodles

close if no further questions.

**Comment by upintheairsheep** [comment]
Just realized that regional exclusivity for certain doodles has not yet been implemented (for realism)
